### ISSUE TITLE
JAWS does not announce popup after column selection

### ISSUE THREAD

**Issue by jeanettetb** [carbon-design-system/carbon#3923]
## Environment
Firefox ESR 60.8.0esr
JAWS Version 2019.1906.10 ILM

## Detailed description

JAWS does not announce popup after column selection. 
![CarbonIssue_JAWS_doesnt_read_popup_after_column_selection](https://user-images.githubusercontent.com/54960530/64378655-279f4100-cffb-11e9-827f-65837d3e62b0.jpg)

**Comment by dakahn** [comment]
Hey, could you fill out the requisite fields in the template provided? Thanks!

**Comment by joshblack** [comment]
Hey there! Going to close this issue as it seems like there hasn't been activity in a while. If you could fill out the issue template, we'd be happy to help address the issue that you're running into!
